response_operations_ui/controllers/survey_controllers.py

```diff
@@ -30,6 +31,7 @@ def get_survey_by_id(survey_id):
 
 
 def get_survey_by_shortname(short_name):
+    short_name = remove_whitespace_from_survey_name(short_name)
```

Comment:
maybe short_name = ''.join(short_name.split()) ?

Reply:
I've changed it to here because it's the only place that does it 👍